src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -1,7 +1,8 @@
 package core.basesyntax.service;
 
+import core.basesyntax.exception.ValidationException;
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws ValidationException;
```

Comment:
```suggestion
    User register(User user);
```
Don’t  throw this exception in the method signature